src/battle_script_commands.c

```diff
@@ -1904,6 +1906,7 @@ s32 CalcCritChanceStage(u8 battlerAtk, u8 battlerDef, u32 move, bool32 recordAbi
 
     return critChance;
 }
+#undef BenefitsFromLeek
```

Comment:
Does it actually work like this? Don't you need to `#undef BENEFITS_FROM_LEEK`? :eyes:

Reply:
Whoops, forgot to change the name at the end ^^;
Fixed!